src/api/app/views/webui/comment/_comment_field.html.haml

```diff
@@ -18,8 +18,12 @@ class: "#{form_method}-comment-form" }, data: { preview_comment_url: preview_com
         class: 'w-100 form-control comment-field'
       .tab-pane.fade{ id: "preview_#{element_suffix}", role: 'tabpanel', 'aria-labelledby': 'preview-comment-tab' }
         .comment-preview.border-bottom.border-gray-300.my-3
-      - case form_method
-      - when :post
-        = f.submit 'Add comment', class: 'btn btn-primary mb-3', data: { disable_with: 'Creating comment...' }, disabled: true
-      - when :put
-        = f.submit 'Update comment', class: 'btn btn-primary mb-3', data: { disable_with: 'Updating comment...' }
+      .comment-controls.mb-3.float-right
```

Comment:
What is the reason for aligning the buttons to the right?

Reply:
It could be discussed for sure. I just wanted to make it more consistent with the other forms where Cancel and Submit buttons are exist. For example:
![delete_project](https://user-images.githubusercontent.com/37581072/99363562-32b75780-28b5-11eb-9259-e22b7746f677.png)
![edit_project](https://user-images.githubusercontent.com/37581072/99363567-33e88480-28b5-11eb-991e-7191dc883feb.png)

As I can see, we do not have one rule for the position of the buttons, and in some cases when only one button is shown, it is aligned by left side. But in all cases with two buttons (at least those, that I've found), we aligning them to the right with the Cancel button first. 

WDYT, should I revert the buttons back to the left side and show them in the order: "Update Comment" "Cancel"?

Reply:
It should be consistent. It's not great that we aren't consistent with that in the whole application. With this change, it just looks weird because the `Reply`, `Edit` and `Delete` buttons are aligned to the left, but then the `Update` and `Cancel` buttons are aligned to the right.

I would revert this change, so keep all the buttons aligned to the left to be consistent on this page.

Reply:
I've reverted the buttons back to the left.